src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,74 @@
+package seedu.duke;
+
+import java.io.IOException;
+
+/**
+ * Main class of the program.
+ */
+public class Duke  {
+    
+    /*Storage object to handle reading and writing to hard disk */
+    private Storage storage;
+    /*Tasklist containing an arraylist of tasks */
+    private TaskList tasks;
+    /*Ui component to handle the user interface of the program */
+    private Ui ui;
+    /*Parser object to parse user input into recognised Commands */
+    private Parser parser;
```

Comment:
```suggestion
    /** Storage object to handle reading and writing to hard disk */
    private Storage storage;
    /** Tasklist containing an arraylist of tasks */
    private TaskList tasks;
    /** Ui component to handle the user interface of the program */
    private Ui ui;
    /** Parser object to parse user input into recognised Commands */
    private Parser parser;
```
I think Javadocs for class members should still start with `/**` according to the coding standard. Might be good to take note?